flow/layers/layer.h

```diff
@@ -145,14 +145,46 @@ class Layer {
 
   bool needs_painting() const { return !paint_bounds_.isEmpty(); }
 
+  // True iff the layer, or some descendant of the layer, performs an
+  // operation which depends on (i.e. must read back from) the surface
+  // on which it is rendered in any way beyond the functionality of
+  // BlendMode. This value has no setter as it is computed from other
+  // flags and properties on the layer.
+  //
+  // see Layer::UpdateTreeReadsSurface()
```

Comment:
I think our style of referencing functions are `|Layer::UpdateTreeReadsSurface|`. CC @chinmaygarde for confirmation.